page.php

```diff
@@ -13,44 +13,98 @@
     <div class="pageContainer">
       <div class="pageLeft">
         <?php if (isset($page['intro'])) : ?>
-          <div class="intro">
-            <?php foreach ($page['intro'] as $p) :
+        <div class="intro">
+          <?php foreach ($page['intro'] as $p) :
               echo '<p>' . $p . '</p>';
             endforeach; ?>
-          </div>
+        </div>
         <?php endif;
 
         if (isset($page['links'])) : ?>
-          <div class="links">
-            <?php foreach ($page['links'] as $link) :
+        <div class="links">
+          <?php foreach ($page['links'] as $link) :
               echo '<a href="' . $link['url'] . '" title="' . $link['text'] . '" class="pageLink" target="_blank">
               ' . $link['text'] . '</a>';
             endforeach; ?>
+        </div>
+        <?php endif;
+
+        if (isset($page['bank_details'])) : ?>
+        <div class="bankDetails">
+          <div>
+            <strong>Bank:</strong> <span id="bank_name"> <?= $page['bank_details']['bank_name']; ?> </span>
+            <button class="copy-bank-btn" onclick="copyToClipboard('bank_name')" title="copy">
```

Comment:
use more descriptive titles like 'copy bank name to clipboard'

Reply:
"bank name" or "bank details"?

Reply:
each button have own title, copy name, copy acc num, copy address etc